src/java/org/apache/cassandra/tools/StandaloneSplitter.java

```diff
@@ -51,7 +52,10 @@
     public static void main(String args[])
     {
         Options options = Options.parseArgs(args);
-        Util.initDatabaseDescriptor();
+        if (Boolean.parseBoolean(System.getProperty(Util.ALLOW_TOOL_REINIT_FOR_TEST, "false")))
```

Comment:
quick question:

how come this line isn't 
```
if (Boolean.getBoolean(Util.ALLOW_TOOL_REINIT_FOR_TEST)))
```

Reply:
Iirc I had to do that bc the tool can execute 'out-of-process', then setting the value of the boolean directly wouldn't work.

Reply:
Mick pinged me about the `get` vs `parse` bit I missed! Will fix it :-) Thx